src/bitrot.py

```diff
@@ -180,12 +179,13 @@ def run(self):
         errors = []
         current_size = 0
         missing_paths = self.select_all_paths(cur)
-        paths, total_size = list_existing_paths(
+        hashes_dict_db = self.select_all_hashes(cur)
+        paths_fs, total_size = list_existing_paths(
```

Comment:
I added the suffixes to differentiate between fs and db info. It was clearer for me to read hashes_db than hashes (and have to remember everytime that they came from the DB), the same reason for paths (do they come from the DB or the FS, ummm? ).